ext_localconf.php

```diff
@@ -140,7 +78,6 @@
         // phpcs:enable
     }
 
-
     if (class_exists(\TYPO3\CMS\Core\Information\Typo3Version::class)) {
```

Comment:
This check is not needed anymore since the `Typo3Version` class is always available now.